ingestion/src/metadata/ingestion/api/topology_runner.py

```diff
@@ -231,7 +231,6 @@ def sink_request(self, stage: NodeStage, entity_request: C) -> Iterable[Entity]:
                 self.update_context(key=stage.context, value=entity)
             if stage.context and stage.cache_all:
                 self.append_context(key=stage.context, value=entity)
-            logger.debug(self.context)
```

Comment:
getting the context after each sink is not giving any meaningful info. Dumping the payload of objects in memory does not add value and it's just too hard to read to extract info from that